workspace/apps/pidp/src/app/features/profile/pages/college-licence/college-licence-declaration/college-licence-declaration.page.ts

```diff
@@ -32,6 +32,11 @@ export class CollegeLicenceDeclarationPage
   public formState: CollegeLicenceDeclarationFormState;
   public colleges: CollegeLookup[];
 
+  public get showNurseValidationInfo(): boolean {
+    const isNurse = this.formState.collegeCode.value === 3;
```

Comment:
I feel like this should be in an enum somewhere, matching on a magic number isn't great.

Also, why do `var = expression; return var` when you could just do `return expression`?

Reply:
1. I'll add an enum
2. Because if you set a breakpoint on line 37 you can inspect the value of isNurse. Memory allocation is cheap so I don't mind the extra line of code if it means potentially easier maintenance/troubleshooting in the future.

Reply:
@james-hollinger I've added the enum. Can you please re-review? Thanks.